go.mod

```diff
@@ -68,6 +68,17 @@ require (
 	k8s.io/utils v0.0.0-20220210201930-3a6ce19ff2f9
 )
 
+require (
```

Comment:
Can you merge this block into the k0s block in the beginning?

Reply:
Done